custom/icds_reports/migrations/sql_templates/update_tables38.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE agg_awc ADD COLUMN num_awcs_conducted_cbe integer DEFAULT 0;
+ALTER TABLE agg_awc ADD COLUMN num_awcs_conducted_vhnd integer DEFAULT 0;
```

Comment:
Don't use DEFAULT as it will lock the entire table in postgres when the migration takes place. These columns should be nullable.

Reply:
Hey hi. I have first used the nullable column only. But realised that such that will raise some not consistent data(calculation will be fine). Meaning, Eg of 'num_awcs_launched', for all the usage form submitted with `add_household=1`  will set `num_awcs_launched=1` and for all the usage for submitted with `add_household=0` will set `num_awcs_launched=0` But for awcs for whom usage forms are not submitted `num_awcs_launched` will be null. should not such column only be 0s or 1s only. I know It does not affect calculation. Asking for consistency purposes. @emord

Reply:
Never Mind. 
Resolved.
btw thanks for telling that. I did not know that Default locks the entire table